### ISSUE TITLE
Additional locality changes for places layer

### ISSUE THREAD

**Issue by nvkelso** [tilezen/vector-datasource#931]
Picking up where https://github.com/tilezen/vector-datasource/issues/840 left off, there are a couple more changes for the "locality" and "region"-alization of the places layer.

1. When we say `state_capital: true`, we should really be saying `region_capital: true`.
2. The Natural Earth YAML lookup for country and region capitals needs to specify `kind: locality` (not `kind: city`).
3. The Naturl Earth YAML lookup for generic `Populated place` should specify `kind: locality` (not `kind: city`).
4. We should also specify the `kind_detail` for Scientific Stations coming from Natural Earth as `scientific_station`.
5. (NEW) When we say `capital: true`, we should really be saying `country_capital: true`.

**Comment by nvkelso** [comment]
We should also specify the `kind_detail` for Scientific Stations coming from Natural Earth as `scientific_station`.

**Comment by nvkelso** [comment]
**tl;dr**

Problem that `region_capital: false,` is showing up in tiles. It should only ever export when it's `true`.

Test case Washington DC (osm ID: 158368533) in http://tile.dev.mapzen.com/vector/v1/all/8/73/97.topojson.

```
properties: {
...
scalerank: 13,
capital: "yes",
admin_level: "2",
population: 646449,
kind: "locality",
name: "Washington",
region_capital: false,
id: 158368533,
...
}
```

===

**Non-consequential:**

Huh, this is producing odd outputs in the debug tool for house styles:

![image](https://cloud.githubusercontent.com/assets/853051/17960142/287e6980-6a59-11e6-9a2c-3fceb33e3233.png)

But the underlying data in [8/41/98.topojson](https://tile.dev.mapzen.com/vector/v1/all/8/41/98.topojson) looks as we expected it to:

```
{
type: "Point",
properties: {
kind: "locality",
scalerank: 13,
name: "Sacramento",
name:lt: "Sakramentas",
kind_detail: "city",
region_capital: true,
source: "openstreetmap.org",
name:ja: "サクラメント",
name:uk: "Сакраменто",
capital: "4",
name:ru: "Сакраменто",
name:ta: "சேக்ரமெண்டோ",
id: 150959789,
population: 453781
},
coordinates: [
618,
800
]
},
```

**Comment by nvkelso** [comment]
WFM
